plugins-bundled/internal/marketplace-app/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Changelog
+
+## Unreleased
+
+Initial release. Not yet fit for production use.
```

Comment:
Needs an update? or removal ;)

Reply:
Have deleted it. Running `yarn watch` from within the marketplace-app folder now gives me compilation warnings about not being able to find the changelog. Doesn't have if running `yarn plugins:build-bundled` command. 🤔